test/src/639-specialfiles/main

```diff
@@ -1,12 +1,36 @@
 cvmfs_test_name="Handling special files"
 
-export CVMFS_TEST_UNIONFS="overlayfs"
-# export CVMFS_IGNORE_SPECIAL_FILES="true"
 test_repo_config="/etc/cvmfs/repositories.d/$CVMFS_TEST_REPO/server.conf"
 
+filename=""
```

Comment:
Sorry, one more thing I overlooked: the global variables in the integration test scripts should be named like

    CVMFS_TEST639_XYZ

this avoids that one test case mistakenly sets another test case's parameter